formicaio/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: formicaio
+name: Formicaio
+tagline: Il potere delle formiche
```

Comment:
I like the tagline, but an english one with more context what the app does would probably be better.